deptofdefense/move.mil
# Fix mis-sized SVGs on IE10/IE11

Fixes #254 .

## Checklist

I have…

- [x] attached screenshots illustrating relevant behavior before and after these changes.

## Summary of Changes

- Restores the `viewBox` attribute to the `<svg>` tag in all SVG files where it had been removed by [778ecef8](https://github.com/deptofdefense/move.mil/commit/778ecef87af214d4b1e722e7174706462d8a599d) and [7ff4e50b](https://github.com/deptofdefense/move.mil/commit/7ff4e50b4577f84ea8a8fbbf671ca3413a00b790)
- Replaces app/assets/images/logos/military-installations.svg and  app/assets/images/logos/military-one-source.svg, which contained only base64-encoded raster image data, with a GIF and a PNG respectively derived from those sites original logos.
- Makes logos on the Resources page a uniform 244 px wide.
- Changes "Defense Travel Regulation" to the correct "Defense **Transportation** Regulation**s**" on the Resources page, and links directly to the relevant Personal Property chapter.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `restore-svg-viewboxes`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load [http://localhost:3000](http://localhost:3000) in Internet Explorer as well as another Web browser of your choice.

## Screenshots

<details><summary>Homepage with service emblems before - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_20_27](https://user-images.githubusercontent.com/26554826/34222163-97c01414-e588-11e7-82a1-d298278b8c63.png)

</details>

<details><summary>Homepage with service emblems after - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_11_58](https://user-images.githubusercontent.com/26554826/34221854-6c9fd63a-e587-11e7-94ed-0d3888e7cef3.png)

</details>

<details><summary>Nightmare Moves before - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_21_37](https://user-images.githubusercontent.com/26554826/34222198-bed387b6-e588-11e7-8be0-59efd615f576.png)

</details>

<details><summary>Nightmare Moves after - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_13_14](https://user-images.githubusercontent.com/26554826/34221915-9b25a35e-e587-11e7-9dd6-a5df0938cd09.png)

</details>

<details><summary>Moving Tips before - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_22_54](https://user-images.githubusercontent.com/26554826/34222252-efc5a778-e588-11e7-9ed7-823af2361fd5.png)

</details>

<details><summary>Moving Tips after - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_15_08](https://user-images.githubusercontent.com/26554826/34221955-d50cd6b4-e587-11e7-802b-cf32978a78b9.png)

</details>

<details><summary>Customer Service (Claims) before - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_16_47](https://user-images.githubusercontent.com/26554826/34222037-265d262c-e588-11e7-96b1-5d0155aa5f22.png)

</details>

<details><summary>Customer Service (Claims) after - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_17_05](https://user-images.githubusercontent.com/26554826/34222040-2b632824-e588-11e7-9ea3-ac6068472ba7.png)

</details>

<details><summary>Service-specific information (US Army) logo before - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_23_51](https://user-images.githubusercontent.com/26554826/34222278-0b90d360-e589-11e7-9f5f-2eb7817fe18e.png)

</details>

<details><summary>Service-specific information (US Army) logo after - IE11</summary>

![virtualbox_ie11 - win7_20_12_2017_13_08_56](https://user-images.githubusercontent.com/26554826/34221701-054d8932-e587-11e7-92ed-50eff2bd6b36.png)

</details>

<details><summary>Resources before - Chrome</summary>

![fireshot capture 66 - tools resources move mil_ - https___move-mil-staging us-east-1](https://user-images.githubusercontent.com/26554826/34222456-9a96a418-e589-11e7-9950-ee3f901340a5.png)

</details>

<details><summary>Resources after - Chrome</summary>

![fireshot capture 65 - tools resources move mil - http___localhost_3000_resources](https://user-images.githubusercontent.com/26554826/34222467-9fe7da86-e589-11e7-9fab-51e0513f92ae.png)

</details>
